src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,76 @@
-import React from 'react';
+import React, { memo, useState } from 'react';
+import cn from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  onSetMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = memo(({ onSetMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+  const [isLoading, setIsloading] = useState(false);
+
+  const handleQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setIsLoadingError(false);
+    setQuery(event.target.value);
+  };
+
+  const submitHandler = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsloading(true);
+
```

Comment:
don't send a request if you have an invalid `title`, just with spaces, for example
Also - put an error like `Put the movie name`